Lil-Peep/src/styles/modules/site-collection.scss

```diff
@@ -0,0 +1,293 @@
+      font-weight: bold;
+      height: 20px;
+      text-align: center;
+      width: 20px;
+
+      &:hover {
+        cursor: pointer;
+      }
+    }
+  }
+}
+
+.current-route {
+  display: none;
+  margin-left: 30px;
+
+  @media #{$breakpoint-tablet} {
+    display: block;
+    font-family: 'Lato', sans-serif;
+    margin-top: 20px;
+
+    .home-route-section {
+      color: #5396e5;
+    }
+
+    .categories-route-section {
+      color: #72808c;
+    }
+  }
+}
+
+.categories-view {
+  font-family: 'Oswald', sans-serif;
+  padding-left: 30px;
+  padding-right: 30px;
+
+  @media #{$breakpoint-mobile} {
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+}
+
+  h1 {
+    font-family: 'Lato', sans-serif;
+    text-align: center;
+  }
+
+.pagination-container {
+  display: flex;
+  justify-content: flex-end;
+  padding-left: 30px;
+  padding-right: 30px;
+
+  @media #{$breakpoint-mobile} {
+    justify-content: space-between;
+  }
+
+  .active-button {
+    color: #636e72;
+    display: inline;
+    font-size: 30px;
+    text-decoration: none;
+
+    @media #{$breakpoint-mobile} {
+      font-size: 20px;
+    }
+  }
+
+  .blocked-button {
+    color: #636e72;
+    display: inline;
+    font-size: 30px;
+    opacity: 0.5;
+    text-decoration: none;
+
+    @media #{$breakpoint-mobile} {
+      font-size: 20px;
+    }
+  }
+
+  .previous {
+    margin-right: 50px;
+    @media #{$breakpoint-mobile} {
+      margin-right: 0px;
+    }
+  }
+}
+
+  .collection-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .product-card-category {
+    background-color: #f5f6fa;
+    flex: 1 0 20%;
+    height: 470px;
+    margin: 10px;
+    max-width: 435px;
+
+    &:hover {
+      background-color: rgba(83, 150, 230, 0.1);
+    }
+
+    .front-label {
+      display: flex;
+      justify-content: flex-end;
+      width: 100%;
+
+      .new-product {
+        background-color: #abc828;
+        border-radius: 3px;
+        color: #ffffff;
+        font-weight: bold;
+        padding: 5px 25px 5px 25px;
+      }
+
+      .not-new-product {
+        color: #f5f6fa;
+        opacity: 1;
+        padding: 5px 25px 5px 25px;
+      }
+    }
+
+    .product-image {
+      align-items: center;
+      display: flex;
+      flex-direction: row;
+
+      height: 300px;
+      justify-content: center;
+
+      img {
+        height: auto;
+        max-height: 300px;
+        max-width: 250px;
+        object-fit: contain;
+      }
+    }
+  }
+
+.product-data {
+  padding-left: 15px;
+  padding-right: 15px;
+
+  p {
+    font-family: 'Lato', sans-serif;
+    margin-bottom: 5px;
+    margin-top: 0;
+  }
+
+  .product-title {
+    color: #636e72;
+    font-weight: bold;
+    text-decoration: none;
+  }
+
+  .price {
+    color: #954180;
+  }
+
+  .offer {
+    display: none;
+  }
+
+  .view-in-detail {
+    background-color: #4a90e2;
+    border-radius: 3px;;
+    border-style: none;
+    padding: 10px 0 10px 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `10px 0` instead of `10px 0 10px 0`